handlers/deploy.go

```diff
@@ -110,12 +128,42 @@ func deploy(ctx context.Context, req types.FunctionDeployment, client *container
 
 	log.Printf("Container ID: %s\tTask ID %s:\tTask PID: %d\t\n", container.ID(), task.ID(), task.Pid())
 
-	ip, netErr := getIP("netns0", int(task.Pid()))
+	id := uuid.New().String()
+	netns := fmt.Sprintf(NetNSPathFmt, task.Pid())
+
+	cni, err := gocni.New(gocni.WithPluginConfDir(CNIConfDir),
+		gocni.WithPluginDir([]string{CNIBinDir}))
+
+	if err != nil {
+		return err
+	}
+
+	// Load the cni configuration
+	if err := cni.Load(gocni.WithLoNetwork, gocni.WithDefaultConf); err != nil {
+		log.Fatalf("Failed to load cni configuration: %v", err)
```

Comment:
Can we do this on start-up and fail there?

Reply:
I'd refactor this and put all CNI initialization code into one handler that just returns the CNI object. I'd just need to pass it to the deployHandler so the network for current container could be created.

Reply:
Yes sounds fine.